habitat-lab/habitat/tasks/rearrange/actions/grip_actions.py

```diff
@@ -100,8 +100,14 @@ def _grasp(self):
         match_coll = None
         contacts = self._sim.get_physics_contact_points()
 
-        robot_id = self._sim.robot.sim_obj.object_id
-        all_gripper_links = list(self._sim.robot.params.gripper_joints)
+        # TODO: the two arguments below should be part of args
+        ee_index = 0
```

Comment:
yes let's make this part of the grasp config. Is the idea that the grasp config is defined per manipulator for a robot with multiple manipulators? It would be good to make sure that the design here makes sense.